CMakeLists.txt

```diff
@@ -302,6 +306,10 @@ if(GENERATE_DOC)
         WORKING_DIRECTORY ${AWE_DOC_DIR}
         DEPENDS ${AWE_SRCS} ${AWE_LUA_FILES} ${AWE_MD_FILES} ${BUILD_DIR}/docs/config.ld
         COMMENT "Generating API documentation${ldoc_desc_suffix}")
+else()
+    add_custom_target(ldoc false
+        COMMENT "ldoc target not generated. Use GENERATE_DOC=1 to enable it, e.g. via `cmake -D GENERATE_DOC=1 build/` or `make distclean ldoc`."
+        VERBATIM)
```

Comment:
I'd still add these two targets to `ALL` (and then have them not just always fail; perhaps just run true?. Since the `man` and `ldoc` targets are included in `ALL`, I guess no one runs them directly anyway.

Reply:
Well, I use `make ldoc` when working on the docs.
This should not silently (although with a comment) pass, without actually building the docs then.